.test.mk

```diff
@@ -123,6 +123,36 @@ test-release-asan: LUAJIT_TEST_ENV = LSAN_OPTIONS=suppressions=${PWD}/asan/lsan.
                                                   unmap_shadow_on_exit=1
 test-release-asan: build run-luajit-test run-test
 
+# Debug ASAN build
+
+.PHONY: test-debug-asan
+test-debug-asan: CMAKE_PARAMS = -DCMAKE_BUILD_TYPE=Debug \
+                                  -DENABLE_WERROR=ON \
```

Comment:
Please align it with the `-DCMAKE_BUILD_TYPE=Debug. It should be one column.

Reply:
Done. See the diff in the comment below about deduplication.